src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1332,4 +1332,17 @@ private function getErrorMessages(): array
             CannotUpdateProductException::class => $this->trans('An error occurred while updating the status for an object.', 'Admin.Notifications.Error'),
         ];
     }
+
+    /**
+     * @return array
+     */
+    private static function getLogDataContext($id_product = null, $error_code = null, $allow_duplicate = null): array
```

Comment:
@PululuK Why using a static ?

Reply:
@Progi1984 because I find it sexy (ᵢ'ₘ ₖᵢddᵢₙg ₒ𝆑 𝄴ₒᵤᵣₛₑ 😁)!
Very good question ... I will correct that ! Thanks man